.github/workflows/build.yml

```diff
@@ -98,10 +92,10 @@ jobs:
           node-version: '12.x'
           registry-url: 'https://registry.npmjs.org'
 
-      - name: Use Python 2.x
+      - name: Use Python 3.x
         uses: actions/setup-python@v2
         with:
-          python-version: '2.x'
+          python-version: '3.x'
```

Comment:
nitpick, but you could set the version in the matrix and reuse that, like done for node.

Reply:
I believe the `matrix` is only available to the `build` (since that is where it is defined).
For `publish` we explicitly set the node version: 

```yaml
- name: Use Node.js 12.x
        uses: actions/setup-node@v1
        with:
          node-version: '12.x'
          registry-url: 'https://registry.npmjs.org'
```

Reply:
Ok fair enough!